Src/ILGPU/Runtime/OpenCL/CLDevice.cs

```diff
@@ -281,28 +281,32 @@ private void InitDeviceInfo()
         /// </summary>
         private void InitGridInfo()
         {
-            // Max grid size
             int workItemDimensions = IntrinsicMath.Max(CurrentAPI.GetDeviceInfo<int>(
-                DeviceId,
-                CLDeviceInfoType.CL_DEVICE_MAX_WORK_ITEM_DIMENSIONS), 3);
-            var workItemSizes = new IntPtr[workItemDimensions];
-            CurrentAPI.GetDeviceInfo(
-                DeviceId,
-                CLDeviceInfoType.CL_DEVICE_MAX_WORK_ITEM_SIZES,
-                workItemSizes);
-            MaxGridSize = new Index3D(
-                workItemSizes[0].ToInt32(),
-                workItemSizes[1].ToInt32(),
-                workItemSizes[2].ToInt32());
+              DeviceId, CLDeviceInfoType.CL_DEVICE_MAX_WORK_ITEM_DIMENSIONS), 3);
```

Comment:
nit: please adjust the indentation to 4 spaces and add a line break after each comma (as in line `294`).